presto-hive/src/main/java/io/prestosql/plugin/hive/util/Statistics.java

```diff
@@ -306,9 +305,6 @@ else if (type.equals(DOUBLE) || type.equals(REAL)) {
         else if (type.equals(DATE)) {
             result.setDateStatistics(new DateStatistics(Optional.empty(), Optional.empty()));
         }
-        else if (type.equals(TIMESTAMP_MILLIS)) {
-            result.setIntegerStatistics(new IntegerStatistics(OptionalLong.empty(), OptionalLong.empty()));
```

Comment:
in a sense, this was valid, right? and OK to keep?

Reply:
The issue is that it will not be ever called is we disable `MIN_VALUE` and `MAX_VALUE` value statistics type for timestamp columns. See `createColumnStatisticsForEmptyPartition` above